_layouts/posts.html

```diff
@@ -15,6 +15,13 @@
 <body>
 <main>
   <p class="no-print"><a href="{{ site.baseurl }}/">< Back to Home</a></p>
+  {% if page.tags %}
+  <div class="tags">
+    {% for tag in page.tags %}
+    <a href='{{ site.baseurl }}/tag/{{tag}}'>#{{tag}}</a>
```

Comment:
Another nitpick: spacing within `{{ tag }}`

These links depend on issue #34. My preference is to remove the anchor tags here and 🚢 this pull, adding the correct anchors in whenever #34 is shipped.